README.md

```diff
@@ -1,4 +1,18 @@
-# fhir-exercises
+# Geting started without installation
+These exercises are implemented using both Python and R. 
+
+## To run Python/Jupyter Notebook without installation
+ - Click [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/stevetsa/fhir-exercises/HEAD)
```

Comment:
This should point to the main repo rather than the fork
```suggestion
 - Click [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/mitre/fhir-exercises/HEAD)
```

Reply:
MyBinder pointing to the main repo rather than the fork